modules/i18n/src/translations/en-US/portals/user.ts

```diff
@@ -81,6 +81,11 @@ export const user: userNS = {
                 header: "Reset password",
                 subheader: "Once you change the password, the user will no longer be able to log in to " +
                     "any application using the current password."
+            },
+            passwordSetZone: {
+                actionTitle: "Set Password",
```

Comment:
How about "Update Password"?

Reply:
This is related to the scenario where te admin tries to set a password for the first time. IMO "Set Password" would be better. WDYT?

Reply:
Had an offline discussion and concluded to proceed with "Set Password"